File: diamond.js
//printing reasult by console
function printhalfdiamond(num){
    for(let i=0 ; i<num ; i++){
        console.log("*".repeat(2*i + 1));
    }
    for(let i=rows-1 ; i>0 ;i--){
        console.log("*".repeat(2*i - 1));
    }
}
//let x = parseInt(prompt("enter ur nb: "));
//printhalfdiamond(rows);

File: diamond.py
#def printhalfdiamond(num):
 #   for i in range(num):
  #      print(" * "*(2*i+1))
   # for i in range(num-1,0,-1):
    #    print(" * "*( 2 *i-1))


def printfulldiamond(num):
    for i in range(num):
        print(" "*(num-i-1) + "*"*(2*i + 1))
    for i in range(num-2,-1,-1):
        print(" "*(num-i-1) + "*"*(2*i + 1))

x =int(input("enter ur nb: "))
#printhalfdiamond(x)
printfulldiamond(x)
